generated/nidigital/nidigital_library_interface.h

```diff
@@ -0,0 +1,143 @@
+//---------------------------------------------------------------------
+// This file is automatically generated. All manual edits will be lost.
+//---------------------------------------------------------------------
+// Library wrapper for implementing interactions with NI-Digital Pattern Driver
+//---------------------------------------------------------------------
+#ifndef NIDIGITAL_GRPC_LIBRARY_WRAPPER_H
+#define NIDIGITAL_GRPC_LIBRARY_WRAPPER_H
+
+#include <grpcpp/grpcpp.h>
+#include <niDigital.h>
```

Comment:
You need to copy copy `niDigital.h` file in imports folder and submit that as part of this PR.

Reply:
the import folder got left out of commit by mistake. fixed that